src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -1,13 +1,22 @@
 (ns status-im2.contexts.chat.messages.content.text.style
   (:require [quo2.foundations.colors :as colors]))
 
+(def spacing-between-blocks 10)
+
+(def parsed-text-block
+  {:transform [{:translateY -4}]})
```

Comment:
could you elaborate why you have to use  transform ?

Reply:
You're right, will replace it with margin